index.html

```diff
@@ -329,5 +325,36 @@ <h2>About me</h2>
         <img class="lastline2" src="images/last line.png" alt="Horizontal-slide">
       </div>
     </section>
+    <section class="modal-section hide" id="modal-section">
+      <div class="modal-contents" id="modal-contents">
+        <!--<div class="title-sect">
+          <h2>Multi Post Stories</h2>
+          <h3>Keeping track of hundreds  of components website</h3>
+          <img src="images/close-icon.png" id="close-projects">
+        </div>
+        <ul class="modal-list-tag">
+          <li class="modal-tags">html</li>
+          <li class="modal-tags">Bootstrap</li>
+          <li class="modal-tags">Ruby on rails</li>
+        </ul>
+        <div class="second-sect">
+          <div class="second-sect-img">
+            <img class="image" src="images/Snapshoot Portfolio.png" alt="Snapshoot image">
+          </div>
```

Comment:
- This image looks good and is visible in the popup of the desktop version but I suggest you reduce the size so that it will fit in the container of the popup as it shows in the Figma design :+1: :sunglasses: 

## Figma design  

![image](https://user-images.githubusercontent.com/95297251/209112165-d2865675-47b2-4b01-a89d-81947431d24f.png)

## Your design 
![image](https://user-images.githubusercontent.com/95297251/209112631-a33a337c-3d9e-4c5d-8c2e-8795e8d90fa3.png)